src/types.md

```diff
@@ -44,16 +45,34 @@ language:
 
 ## Numeric types
 
+### Boolean type
```

Comment:
Boolean isn't a numeric type, so it shouldn't be inside the "Numeric types" section. Move it, and it'll be good.